Frontend/src/api/UploadPaper.jsx

```
@@ -0,0 +1,124 @@
+        headers: { "Content-Type": "multipart/form-data" },
+      });
+
+      alert("Paper uploaded successfully!");
+      setTitle("");
+      setAuthor("");
+      setResearchField("");
+      setFile(null);
+      navigate("/dashboard/research/my_post_research_papers"); // ✅ Redirect after upload
+    } catch (error) {
+      alert("Upload failed: " + JSON.stringify(error.response?.data || error.message));
+    } finally {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)